package.json

```
@@ -0,0 +1,36 @@
+{
+  "name": "mdlinks",
+  "version": "0.0.1",
+  "description": "libreria de markdown",
```

Recomiendo mejorar tu descripción y que fuese en ingles
```suggestion
  "description": "library for read md files by Adriana",
```